src/test/java/moimoi/util/command/AddCommandTest.java

```diff
@@ -0,0 +1,169 @@
+package moimoi.util.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+import moimoi.util.Storage;
+import moimoi.util.TaskList;
+import moimoi.util.exception.InvalidDateTimeException;
+import moimoi.util.exception.MissingArgumentException;
+import moimoi.util.exception.MoiMoiException;
+import moimoi.util.task.Task;
+
+public class AddCommandTest {
+
+    private Storage storage = new Storage("data/test.txt");
+    private TaskList tasks = new TaskList(new ArrayList<Task>());
+
+    @Test
+    public void testisExit() {
```

Comment:
Would testIsExit be the correct naming standard here?

Reply:
Yes, seems like I overlooked this part 😆 Thanks for pointing it out!